db/migrate/20170512042542_create_categories.rb

```diff
@@ -0,0 +1,12 @@
+class CreateCategories < ActiveRecord::Migration
+  def change
+    create_table :categories do |t|
+      t.string :canonical_title, required: true, null: false
```

Comment:
Yeah, didn't realize we were using it differently than anidb was at first, so hopefully this will avoid some confusion in the future